app/brave_settings_strings.grdp

```diff
@@ -476,6 +476,12 @@
   <message name="IDS_SETTINGS_BRAVE_SHIELDS_COOKIE_CONTROL_LABEL" desc="Default Brave cookies control setting label">
     Block cookies
   </message>
+  <message name="IDS_SETTINGS_BRAVE_SHIELDS_COOKIE_CONTROL_BLOCK_ALL_DEPRECATED_LABEL" desc="Block all cookies deprecation warning">
+    Block all cookies option is deprecated. To force-enable, 
+  </message>
+  <message name="IDS_SETTINGS_BRAVE_SHIELDS_COOKIE_CONTROL_BLOCK_ALL_DEPRECATED_LINK_LABEL" desc="Block all cookies learn more link text">
+    Learn more
+  </message>
```

Comment:
The `To force-enable, Learn more` part doesn't look right to me. Also, because this warning is shown with the setting itself, I don't think repeating `Block all cookies` is necessary. I would suggest perhaps:

```
This option is deprecated. <link>Learn how to force-enable</link>
```
cc: @rmcfadden3 WDYT?

Reply:
@mkarolin how about `The "Block all Cookies" option has been deprecated. To learn how to force-enable this option, visit [our help page](https://support.brave.com/).`

Note that I just used the general support.brave.com link as a placeholder on the href. We should obviously embed the exact URL once we have it.

If the goal is to send someone to Flags or something, rather than help center, LMK and I can tweak the copy accordingly.

Reply:
Thanks @rmcfadden3! Sounds good to me.